universal-login-react/src/ui/styles/waitingFor.sass

```diff
@@ -1,64 +1,65 @@
 .universal-login-waiting-for
-  max-width: 570px
-  margin: 0 auto
-  padding-top: 120px
-  min-height: 600px
+  .universal-login-default
```

Comment:
I don't think that all the styles should be in `.universal-login-defaults`. Colors should be defaults for sure, but height, width, etc. should be in `.universal-login-waiting-for` class, not only in `.universal-login-default` class. 

The way you did it `.universal-login-waiting-for` class seems to be redundant.